Formula/rxvt-unicode.rb

```diff
@@ -39,6 +45,12 @@ class RxvtUnicode < Formula
   end
 
   def install
+    resource("libptytty").stage do
+      system "cmake", ".", "-DCMAKE_INSTALL_PREFIX=#{libexec}", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
+      system "make", "install"
+    end
+    ENV.prepend_path "PATH", libexec/"lib"
```

Comment:
need some help to understand the PATH setup for this resource.

Reply:
That doesn't look right. You probably want `libexec/"bin"`.